src/main/rx/spektrum.h

```diff
@@ -20,5 +20,5 @@
 #define SPEKTRUM_SAT_BIND_DISABLED 0
 #define SPEKTRUM_SAT_BIND_MAX 10
 
-uint8_t spektrumFrameStatus(void);
```

Comment:
@MJ666 @proggod 
I dont disagree. For those that have working systems, there may not be much of a point in this change. There are a few things that this change brings for existing and new users.

New users:
The biggest issue is with the vortex and vortex pro (for those using serial rx's). There have been issues with people that own DX6i's not being able to arm or enter the osd menu due to travel setting limitation being 125% (Immersionrc recommends 148% travel). This requires users to add a rudder to rudder mix to get the additional travel needed to arm, which starts making set up a bit tedious. @proggod this probably would be fixed by endpoint configuration as well.

Existing users:
-The main advantage for this would be for users transitioning from ppm/pwm to serial. Due to cleanflights current method of interpreting spektrum serial data, there is an inconsitency between the RC command values given through ppm/pwm vs serial (assuming use of the same transmitter model/travel). This would mean people having to re-adjust travel when switching from ppm to serial. It could be considered a minor inconvenience, but its an inconvenience that could be removed. It makes things easier to explain as well when you only need 1 model setup regardless of whether you're using a ppm or serial receiver.
-Yes I agree, this change would be more tedious for existing users due to having to reduce travel by 25%, or reducing rc rates in cleanflight (travel adjustment is probably easier).

Also with potential future changes to how cleanflight uses rc channel data (float instead of int maybe?), this change would take advantage of the 2048 resolution provided by the recevier rather than having it chopped in half the way it currently does :grin:. I'm not very familiar with cleanflight yet though and am not sure how viable this sort of change might be.

Overall, the main goal of this change is to fix something that was initially implemented incorrectly (though functional) to provide consistency with other spektrum products. If this is still something we think isn't worth changing at the moment, maybe it could be re considered when a stick calibration procedure is introduced to cleanflight. Then this change wouldn't affect users since they'll have to calibrate inputs anyway.
And of course, the addition of frame loss data which previously was ignored in the data stream. This could be used for telemetry or osd display. This wouldn't affect existing users setup in any way and could possibly be merged separately.